
arm_configurations:
  side_tuck:
    position:
      right_arm: [ -2.1, 1.26, -1.8, -1.9, 3.5, -1.8, 1.5708]
      left_arm:  [ 2.1, 1.26, 1.8, -1.9, -3.5, -1.8, 1.5708]
    trajectory:
      right_arm: [[-0.968, 0.129, -0.554, -1.891, -1.786, -1.127, 0.501], [-2.135, -0.02, -1.64, -2.07, -1.64, -1.68, 1.398], [-2.1, 1.26, -1.8, -1.9, 3.5, -1.8, 1.5708]]
      left_arm: [[0.968, 0.129, 0.554, -1.891, 1.786, -1.127, 0.501], [2.135, -0.02, 1.64, -2.07, 1.64, -1.68, 1.398], [2.1, 1.26, 1.8, -1.9, -3.5, -1.8, 1.5708]]

  side_carry:
    position:
      right_arm: [ -2.1, 1.23, -2.06, -1.69, 0.3, -1.32, 1.5708]
      left_arm:  [ 2.1, 1.23, 2.06, -1.69, -0.3, -1.32, 1.5708]
    trajectory:
      right_arm: [[-0.968, 0.129, -0.554, -1.891, -1.786, -1.127, 0.501], [-2.135, -0.02, -1.64, -2.07, -1.64, -1.68, 1.398], [-2.1, 1.23, -2.06, -1.69, 0.3, -1.32, 1.5708]]
      left_arm: [[0.968, 0.129, 0.554, -1.891, 1.786, -1.127, 0.501], [2.135, -0.02, 1.64, -2.07, 1.64, -1.68, 1.398], [2.1, 1.23, 2.06, -1.69, -0.3, -1.32, 1.5708]]

